apps/src/code-studio/components/progress/UnitOverviewHeader.jsx

```diff
@@ -127,17 +121,6 @@ class UnitOverviewHeader extends Component {
       versionWarningDetails = i18n.wrongCourseVersionWarningDetails();
     }
 
-    // Only display viewable versions in script version dropdown.
-    const filteredVersions = versions.filter(version => version.canViewVersion);
```

Comment:
Why don't we need to filter these anymore?

Reply:
We do it all on the backend before we pass the information down as part of `assignable_course_offerings_info`